lib/evaluator.ex

```diff
@@ -1,6 +1,6 @@
 defmodule Veritaserum.Evaluator do
   @moduledoc """
-  Evaluats words, boosters, negators and emoticons.
+  Evaluats words, boosters, negators, and emoticons.
```

Comment:
this is not correct though 
```suggestion
  Evaluats words, boosters, negators and emoticons.
```